build-system/tasks/runtime-test/helpers.js

```diff
@@ -377,6 +383,7 @@ async function createKarmaServer(
     .on('browsers_ready', () => karmaBrowsersReady())
     .on('browser_start', browser => karmaBrowserStart(browser))
     .on('browser_complete', browser => karmaBrowserComplete(browser))
+    .on('browser_error', (browser, error) => karmaBrowserError(browser, error))
```

Comment:
Out of curiosity, why do we pass a `() => callbackFn()` instead of just `callbackFn` in this file?